### PR TITLE
🔧(*/flavors) refactor and clean settings

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

I was going to and from our deployment projects and this repository to fix settings. But then I realized there were major problems and it would be faster to refactor everything at once with some methodology.

## Proposal

I went through the yaml files that were added to the dogwood and eucalyptus projects, and did the following:

- remove settings that were not relevant to lms resp. cms,
- remove settings that are unlikely to be set per project and make sure
  they are properly set in docker_run_*.py files,
- add config methods in docker_run_*.py files for settings that are
  likely to be set per project.

### CODE REVIEW

File: releases/dogwood/3/fun/config/cms/fun.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 
 SITE_VARIANT = "cms"
 
+PAYMENT_ADMIN = "paybox@fun-mooc.fr"
+FAVICON_PATH: fun / images / favicon.ico
```
Comment (jmaupetit) [inline releases/dogwood/3/fun/config/cms/fun.py]:
```suggestion
FAVICON_PATH = "fun/images/favicon.ico"
```

File: releases/dogwood/3/bare/config/lms/docker_run_production.py [hunk 2]
```diff
@@ -397,6 +399,8 @@
 META_UNIVERSITIES = config("META_UNIVERSITIES", default={}, formatter=json.loads)
 COMMENTS_SERVICE_URL = config("COMMENTS_SERVICE_URL", default="")
 COMMENTS_SERVICE_KEY = config("COMMENTS_SERVICE_KEY", default="")
+CERT_NAME_SHORT = config("CERT_NAME_SHORT", default=CERT_NAME_SHORT)
+CERT_NAME_LONG = config("CERT_NAME_LONG", default=CERT_NAME_LONG)
```
Comment (jmaupetit) [inline releases/dogwood/3/bare/config/lms/docker_run_production.py]: Those are not related to fun-apps?

File: releases/dogwood/3/fun/config/lms/docker_run_production.py [hunk 3]
```diff
@@ -431,6 +431,8 @@
 META_UNIVERSITIES = config("META_UNIVERSITIES", default={}, formatter=json.loads)
 COMMENTS_SERVICE_URL = config("COMMENTS_SERVICE_URL", default="")
 COMMENTS_SERVICE_KEY = config("COMMENTS_SERVICE_KEY", default="")
+CERT_NAME_SHORT = config("CERT_NAME_SHORT", default=CERT_NAME_SHORT)
+CERT_NAME_LONG = config("CERT_NAME_LONG", default=CERT_NAME_LONG)
```
Comment (jmaupetit) [inline releases/dogwood/3/fun/config/lms/docker_run_production.py]: ditto?

File: releases/dogwood/3/fun/config/lms/fun.py [hunk 4]
```diff
@@ -160,6 +160,7 @@
 
 FUN_SMALL_LOGO_RELATIVE_PATH = "funsite/images/logos/funmooc173.png"
 FUN_BIG_LOGO_RELATIVE_PATH = "funsite/images/logos/funmoocfp.png"
+FAVICON_PATH: fun / images / favicon.ico
```
Comment (jmaupetit) [inline releases/dogwood/3/fun/config/lms/fun.py]:
```suggestion
FAVICON_PATH = "fun/images/favicon.ico"
```

File: releases/dogwood/3/fun/config/lms/fun.py [hunk 5]
```diff
@@ -202,7 +203,8 @@
         "crop": "smart",
     },  # https://developers.facebook.com/docs/sharing/best-practices
 }
-
+THUMBNAIL_PRESERVE_EXTENSIONS = True
+THUMBNAIL_EXTENSION = "png"
```
Comment (jmaupetit) [inline releases/dogwood/3/fun/config/lms/fun.py]: Sure this is related to fun-apps?

File: releases/eucalyptus/3/bare/config/lms/docker_run_production.py [hunk 6]
```diff
@@ -448,6 +450,8 @@
 META_UNIVERSITIES = config("META_UNIVERSITIES", default={}, formatter=json.loads)
 COMMENTS_SERVICE_URL = config("COMMENTS_SERVICE_URL", default="")
 COMMENTS_SERVICE_KEY = config("COMMENTS_SERVICE_KEY", default="")
+CERT_NAME_SHORT = config("CERT_NAME_SHORT", default=CERT_NAME_SHORT)
+CERT_NAME_LONG = config("CERT_NAME_LONG", default=CERT_NAME_LONG)
```
Comment (jmaupetit) [inline releases/eucalyptus/3/bare/config/lms/docker_run_production.py]: ditto?

File: releases/eucalyptus/3/wb/config/lms/docker_run_production.py [hunk 7]
```diff
@@ -481,6 +483,8 @@
 META_UNIVERSITIES = config("META_UNIVERSITIES", default={}, formatter=json.loads)
 COMMENTS_SERVICE_URL = config("COMMENTS_SERVICE_URL", default="")
 COMMENTS_SERVICE_KEY = config("COMMENTS_SERVICE_KEY", default="")
+CERT_NAME_SHORT = config("CERT_NAME_SHORT", default=CERT_NAME_SHORT)
+CERT_NAME_LONG = config("CERT_NAME_LONG", default=CERT_NAME_LONG)
```
Comment (jmaupetit) [inline releases/eucalyptus/3/wb/config/lms/docker_run_production.py]: ditto?

File: releases/dogwood/3/fun/config/lms/fun.py (hunk 5 above)
Comment (sampaccoud) [reply releases/dogwood/3/fun/config/lms/fun.py]: Yes sure. This is used by the course fun-app to compute thumbnails via easy thumbnails.

A search on edx-plateform confirms that is is not unknown: https://github.com/edx/edx-platform/search?q=THUMBNAIL_EXTENSION&unscoped_q=THUMBNAIL_EXTENSION

File: releases/dogwood/3/bare/config/lms/docker_run_production.py (hunk 2 above)
Comment (sampaccoud) [reply releases/dogwood/3/bare/config/lms/docker_run_production.py]: No you can also find them in openedx's code base: https://github.com/edx/edx-platform/search?q=CERT_NAME_SHORT&unscoped_q=CERT_NAME_SHORT

**Comment by sampaccoud** [comment]
@jmaupetit I handled your comments and added a release for `dogwood.3-fun` and `eucalyptus.3-wb`.
